scripts/README.md

```diff
@@ -0,0 +1,58 @@
+# Scripts
+
+## Benchmarking on sift1m
+
+### Download dataset
+
+```shell
+wget http://ann-benchmarks.com/sift-128-euclidean.hdf5
+```
+
+### Run storage backends
+
+```shell
+docker-compose up -d
+```
+
+### Run benchmarking script
+
+Install dependencies:
+
+```
+pip install "docarray[common,benchmark,qdrant,annlite,weaviate,elasticsearch,redis]"
+```
+
+Run the benchmark on [sift1m](https://www.tensorflow.org/datasets/catalog/sift1m) or random dataset:
+
+```
+# sif1M
+python benchmarking_sift1m.py [-h] [--exclude-backends EXCLUDE_BACKENDS] 
+
+# random
+python benchmarking.py [-h] [--fixed-hnsw] [--exclude-backends EXCLUDE_BACKENDS] 
```

Comment:
btw when I tried python benchmarking.py --help it looks like the script is loading data, can you check this ?